.ci/update-github-page.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -e
+
+checkForVariable() {
+  VAR_NAME=$1
+  if [ ! -v "$VAR_NAME" ]; then
+    echo "Error: Define $1 environment variable"
+    exit 1
+  fi
+}
+
+checkForVariable "READ_ONLY_TOKEN"
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+cd .ci-temp/checkstyle
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name")
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
```

Comment:
release notes update happening after release is done so if we do release of 10.3.1 -> 10.3.2
both such releases already done and exists at https://api.github.com/repos/checkstyle/checkstyle/releases 
so you can take two latest json objects and take `tag_name` from them and it will be your CS_RELEASE_VERSION and LATEST_RELEASE_TAG

please do this.

Reply:
one more time look at https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams.md

this scripts should be executed at this area
![image](https://user-images.githubusercontent.com/812984/181917860-c1f8b2e6-1ac4-4f16-8b78-618599d59e7a.png)

Reply:
![image](https://user-images.githubusercontent.com/812984/181918016-b4b3f884-0898-4921-899b-75fabed22906.png)

Reply:
@romani DO you want me to use LATEST_RELEASE_TAG as `NEW RELEASE ` while updating github page

Reply:
you should have two  variables
`LATEST_TAG`
and 
`PREVIOUS_TAG`

Reply:
Working on it.

Reply:
Is it done ?

Reply:
Please check comment in discord.

Reply:
URL give you access to all releases order by when they are created.
You need to write logic that takes target release out all, and previous release.
From such Json objects you should take tag name and it will be your start and end sha reference

Reply:
Try to get index of release (10.3.1)
https://stackoverflow.com/questions/53986312/have-jq-return-the-index-number-of-an-element-in-an-array


After that do mathematical reduction on index by 1. And then take previous release (10.3) , simply by index ( 'index of 10.3.1' - 1).

Reply:
@romani I do:

```
LATEST_RELEASE_TAG=$( curl -s https://api.github.com/repos/checkstyle/checkstyle/releases \
| jq '[.[].tag_name] | to_entries | .[] | select(.value=="checkstyle-10.3.1") | .key')

echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
```
![Screenshot from 2022-08-16 21-28-21](https://user-images.githubusercontent.com/71710042/184925771-fb37d478-b9ba-4639-8547-3a2f5b32aa51.png)

Reply:
It is not a tag, it is key.
But yes direction is correct.

Please share commands and output to print tag name of release 10.3.1 and its previous.

Reply:
```
github_rsp="$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases?per_page=2)"

IFS=";" read -r LATEST_TAG PREVIOUS_TAG < <(jq -r '[.[].tag_name] | join(";")' <<< "$github_rsp")

echo "Latest version: $LATEST_TAG"
echo "Previous version: $PREVIOUS_TAG"

```

Reply:
@romani Please help me with understanding where I will have to use the key. and how can I do mathematical reduction?

Reply:
TARGET_RELASE_KEY=TARGET_RELASE_KEY-1

Reply:
We need +1 as releases are ordered descending

Reply:
see working code at https://github.com/checkstyle/checkstyle/pull/12007#issuecomment-1217268954